beacon-chain/state/stategen/replay.go

```diff
@@ -308,18 +308,36 @@ func (s *State) genesisRoot(ctx context.Context) ([32]byte, error) {
 	return stateutil.BlockRoot(b.Block)
 }
 
-// This retrieves the archived root in the DB.
-func (s *State) archivedRoot(ctx context.Context, slot uint64) [32]byte {
+// This retrieves the highest archived root that represents the input slot in the DB.
+func (s *State) archivedRoot(ctx context.Context, slot uint64) ([32]byte, error) {
 	archivedIndex := uint64(0)
 	if slot/params.BeaconConfig().SlotsPerArchivedPoint > 1 {
 		archivedIndex = slot/params.BeaconConfig().SlotsPerArchivedPoint - 1
 	}
-	return s.beaconDB.ArchivedPointRoot(ctx, archivedIndex)
+	fmt.Println(archivedIndex)
```

Comment:
I see this number increasing constantly, reached 130 and I turned off my slasher.

Reply:
That's good. That means your slasher is doing work 👍 I removed the debug log